Cargo.toml

```diff
@@ -72,7 +72,7 @@ members = [
   "wallet/wallet-rpc-lib",              # Wallet RPC definitions library.
   "wallet/wallet-test-node",            # Node for wallet testing as a library.
   "wasm-wrappers",                      # WASM wrappers for various components.
-  "wasm-wrappers/wasm-doc-gen",         # WASM wrappers documentation generator.
```

Comment:
This should be one entry per line.

Reply:
Please elaborate. I'm not sure I understand the complaint.